README.md

```diff
@@ -65,58 +67,109 @@ This step should be done in the case that changes have been made to any of the y
+```
 
 ### Creating a bundle image
-A bundle image can be created by editing the CSV in deploy/bundle/windows-machine-config-operator/manifests/
+You can skip this step if you want to run the operator locally with OLM. See [Running Operator Locally with OLM](#running-locally-with-olm)
+
+A bundle image can be created by editing the CSV in deploy/olm-catalog/windows-machine-config-operator/manifests/
 and replacing `REPLACE_IMAGE` with the location of the WMCO operator image you wish to deploy.
 See [the build instructions](#build) for more information on building the image.
 
 You can then run the following command in the root of this git repository
 ```shell script
-operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/olm-catalog/windows-machine-config-operator/0.0.0 \
---channels alpha --default-channel alpha --image-builder podman
+operator-sdk bundle create $BUNDLE_REPOSITORY:$BUNDLE_TAG -d deploy/olm-catalog/windows-machine-config-operator/manifests \
+--channels alpha --default-channel alpha
 ```
 The variables in the command should be changed to match the container image repository you wish to store the bundle in.
 You can also change the channels based on the release status of the operator.
+This command should create a new bundle image. Bundle image and operator image are two different images. 
 
-You should then push the image to the remote repository
+You should then push the newly created bundle image to the remote repository
 ```shell script
-podman push $BUNDLE_REPOSITORY:$BUNDLE_VERSION
+podman push $BUNDLE_REPOSITORY:$BUNDLE_TAG
 ```
 
 You should verify that the new bundle is valid:
 ```shell script
-operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_VERSION --image-builder podman
+operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_TAG
 ```
 
 ### Creating a new operator index
 An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
 cluster.
 
 ```shell script
-opm index add --bundles $BUNDLE_REPOSITORY:$VERSION_TAG --tag $INDEX_VERSION
+opm index add --bundles $BUNDLE_REPOSITORY:$BUNDLE_TAG --tag $INDEX_REPOSITORY:$INDEX_TAG
+```
+
+You should then push the newly created index image to the remote repository
+```shell script
+podman push $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
 #### Editing an existing operator index
 An existing operator index can have bundles added to it:
 ```shell script
-opm index add --from-index $INDEX_VERSION
+opm index add --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
 and removed from it:
 ```shell script
-opm index rm --from-index $INDEX_VERSION
+opm index rm --from-index $INDEX_REPOSITORY:$INDEX_TAG
 ```
 
 ### Deploying the operator on a local cluster
 #### Openshift Console
 This deployment method is currently not supported. Please use the [CLI](#cli)
 
+#### Create Secrets
+Create the cloud provider and cloud private key secrets. The cloud-private-key should match the keypair used in the
```

Comment:
Please mention that this needs to be done for running the operator locally without creating the bundle images and when using the CLI

Reply:
As this is required step, moved this inside running locally using olm and created 2 sub sections i.e. `running with bundle and index images` and `running without bundle and index images`